BookService/Logic/ILibrary.cs

```diff
@@ -0,0 +1,189 @@
+﻿using Logic.Models;
+using System;
+using System.Collections.Generic;
+
+namespace Logic
+{
+    /// <summary>
+    /// Interface of the book's collecton
+    /// </summary>
+    public interface ILibrary
+    {
+
+        /// <summary>
+        /// Gets size of the book's collection
+        /// </summary>
+        int BooksCount { get; }
+
+        /// <summary>
+        /// Gets size of the authors's collection
+        /// </summary>
+        int AuthorsCount { get; }
+
+        /// <summary>
+        /// Gets size of the genre's collection
+        /// </summary>
+        int GenresCount { get; }
+
+        #region Author Methods
+
+        /// <summary>
+        /// Get author value by it's index
+        /// </summary>
+        /// <param name="id">Index of the selected author</param>
+        /// <returns>author by selected index</returns>
+        /// <exception cref="ArgumentOutOfRangeException">Throw when index out of
```

Comment:
Personally me prefer put this message in one line. As it is not code, it`s not required to show it on one screen if screen is small.